src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompiler.cs

```diff
@@ -180,13 +180,20 @@ internal abstract class RegexCompiler
         /// <summary>A macro for _ilg.Emit(OpCodes.Ldarg_0).</summary>
         protected void Ldthis() => _ilg!.Emit(OpCodes.Ldarg_0);
 
+        /// <summary>A macro for _ilgEmit(OpCodes.Ldarg_1) </summary>
+        private void Ldarg_1() => _ilg!.Emit(OpCodes.Ldarg_1);
```

Comment:
Subsequently I'm really tempted to go and delete all of these "macros".  I don't know that they bring a lot of value, and frequently evolving the code means we end up adding and removing them based on the exact opcodes required.

Reply:
In my opinion they do tend to make the code slightly easier to read, and also it makes our code look closer to the IL on a regular assembly so I don't mind them. I'm not opposed to removing them either, but I wouldn't be opposed to keep them either.